src/main/java/edu/greenblitz/robotName/utils/GBCircle.java

```diff
@@ -0,0 +1,93 @@
+package edu.greenblitz.robotName.utils;
+
+import edu.wpi.first.math.geometry.Translation2d;
+
+/**
+ * A circle.
+ * A radius and center position, EPSILON for when the given point is right below the center (Cx == Px)
+ */
+public class GBCircle {
+    private Translation2d centerPosition;
+    private double radius;
+    private final double EPSILON = 0.1;
```

Comment:
if this is the same for every GBCircle, shouldn't it be static?

Reply:
I think it should be default value and have also a constructur that gets the epsilon.